<launch>
<!-- Launch file for ouster OS2-64 LiDAR -->

    <arg name="rviz" default="true" />

    <rosparam command="load" file="$(find lt_lio)/config/mid360_loc_FFR.yaml" />

    <node pkg="lt_lio" type="ltlio_main" name="laserMapping" output="screen" /> 

    <group if="$(arg rviz)">
    <node launch-prefix="nice" pkg="rviz" type="rviz" name="rviz" args="-d $(find lt_lio)/config/loc.rviz" />
    </group>

</launch>
